.circleci/config/workflows/build-test.yml

```diff
@@ -7,69 +7,78 @@ jobs:
           ignore:
             - stable-website
             - /^.-ui\b.*/
+
   - lint-go:
-      filters: &backend_branches_filter
+      # check branches are almost all the backend branches
+      filters: &backend_check_branches_filter
         branches:
           ignore:
             - /^.-ui\b.*/
             - /^docs-.*/
             - stable-website
-
   - build-darwin-binaries:
-      filters: *backend_branches_filter
-  - test-windows:
-      filters: *backend_branches_filter
+      filters: *backend_check_branches_filter
+  - test-e2e:
```

Comment:
Currently this only runs the single "integration test" (testing vault API compatibility matrix); as I work on other E2E work this is likely to get folded into the framework we run on nightly, at which point we can eliminate it here.